### PR TITLE
Feature/site wide footer

### PR DESCRIPTION
**Pull request by paulmsmith**
This is the code to add a site-wide footer to the app. Statically for the time being. Links and content to be updated in a separate branch as part of a different ticket.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
 // ############################################################################
 @import "helpers/all";
 @import "tools/all";
+@import "core/all";
 @import "components/all";
-@import "components/direct_upload";
```
Comment (paulmsmith) [inline app/assets/stylesheets/application.scss]: I moved this to be in the components 'all' file

File: app/assets/stylesheets/core/_template.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.nlhf-template {
```
Comment (paulmsmith) [inline app/assets/stylesheets/core/_template.scss]: This is what govuk do to handle short pages.

File: app/assets/images/nhmf_logo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...9c-0.493,-1.231 -1.281,-2.462 -2.464,-3.301c0.296,0.986 0.936,2.414 0.344,3.647c-0.985,2.118 -3.15,2.906 -5.22,3.202c-2.956,0.197 -5.861,-0.837 -7.191,-3.645c-0.79,-1.677 -1.084,-4.188 0.244,-5.716c4.729,-5.023 11.307,-0.726 16.651,0.247c5.362,0.977 12.02,2.957 16.894,-0.344l3.068,-1.959c-7.637,8.412 -21.342,13.338 -22.919,25.357c-0.294,2.759 0.247,5.419 2.267,7.339c-8.177,-5.517 -14.187,-13.792 -16.895,-22.462m-1.577,-6.01c-1.675,-8.621 -0.295,-17.782 2.955,-25.22c-0.788,4.532 0.839,8.818 4.631,11.477c1.772,1.232 4.039,1.33 6.157,0.739c2.07,-0.64 3.793,-2.857 3.989,-4.975c-0.047,-0.788 0.199,-1.331 -0.097,-1.921c-0.493,1.083 -1.33,2.808 -2.858,2.955c-3.299,0.543 -6.107,-0.985 -8.372,-3.399c-1.33,-1.674 -1.923,-3.941 -1.283,-6.159c0.396,-1.081 1.134,-2.362 2.365,-2.855c4.652,-1.591 7.678,-0.666 10.245,2.855c1.935,3.044 2.808,5.765 4.729,8.377c3.399,6.403 9.52,11.895 17.106,11.847c-6.799,2.462 -14.481,0.43 -20.751,-1.996c-6.847,-2.192 -13.694,-1.774 -17.633,3.743c-0.788,1.33 -1.183,2.858 -1.183,4.532m43.249,-56.647c-26.931,0 -48.764,21.833 -48.764,48.766c0,26.931 21.833,48.764 48.764,48.764c26.931,0 48.764,-21.833 48.764,-48.764c0,-26.933 -21.833,-48.766 -48.764,-48.766"/></g></svg>
```
Comment (stuartmccoll) [inline app/assets/images/nhmf_logo.svg]: Are we still planning to use this later?

Comment (paulmsmith) [reply app/assets/images/nhmf_logo.svg]: Hmm. We might. I have a local copy so will delete it for now.

**Pull request merged**